spec/features/create_object_h2_spec.rb

```diff
@@ -59,7 +62,6 @@
     click_button 'Continue'
 
     # Work Deposit view
-    fill_in 'What\'s changing?', with: 'Initial metadata'
```

Comment:
this was a bug introduced -- the PR for reserve a purl was old, so I guess this changed.

Reply:
Yeah, this is no longer shown on the first version